Lib/defcon/objects/glyph.py

```diff
@@ -804,6 +804,10 @@ def instantiateAnchor(self, anchorDict=None):
             glyph=self,
             anchorDict=anchorDict
         )
+        if anchor.identifier is not None:
+            identifiers = self._identifiers
+            assert anchor.identifier not in identifiers
+            identifiers.add(anchor.identifier)
```

Comment:
hm. is it a good idea to add the anchor's identifier to `self._identifiers` in the `instantiateAnchor` method? I'd expect that to occur in the `insertAnchor`.

Reply:
good point, my oversight.